draft-ietf-lamps-rfc4210bis.md

```diff
@@ -1170,130 +1182,100 @@ other things) the certificate containing the public key of the
 
-Note: Instead of using a repository, the end entity can use the root CA update
-general message to request the respective certificates from a PKI management
-entity, see {{sect-5.3.19.15}}, and follow the required validation steps.
 
-#### Verification in Cases 1, 4, 5, and 8
+#### Verification in Cases 1 and 4
 {: id="sect-4.4.2.1"}
 
 In these cases, the verifier has a local copy of the CA public key
 that can be used to verify the certificate directly.  This is the
 same as the situation where no key change has occurred.
 
-Note that case 8 may arise between the time when the CA operator has
-generated the new key pair and the time when the CA operator stores
-the updated attributes in the repository.  Case 5 can only arise if
-the CA operator has issued both the signer's and verifier's
-certificates during this "gap" (the CA operator SHOULD avoid this as
-it leads to the failure cases described below)
-
 
 #### Verification in Case 2
 {: id="sect-4.4.2.2"}
 
-In case 2, the verifier must get access to the old public key of the
-CA.  The verifier does the following:
+In case 2, the verifier must get access to the new public key of the
+CA. Case 2 will arise when the CA operator has issued the verifier's
+certificate, then changed the CA's key, and then issued the signer's
+certificate; so it is quite a typical case.
 
+The verifier does the following:
 
-1. Look up the caCertificate attribute in the repository and pick
-  the OldWithNew certificate (determined based on validity periods;
-  note that the subject and issuer fields must match);
 
-1. Verify that this is correct using the new CA key (which the
-  verifier has locally);
+1. Get the "new with new" and "new with old" certificates. The
+  location to retrieve theses certificates from, may be available in
+  the authority information access extension of the "old with old"
+  certificate, see caIssuers access method in Section 4.2.2.1 of
+  {{RFC5280}}, or it may be locally configured.
 
-1. If correct, check the signer's certificate using the old CA key.
 
-Case 2 will arise when the CA operator has issued the signer's
-certificate, then changed the key, and then issued the verifier's
-certificate; so it is quite a typical case.
 
+    1. If a repository is available, look up the certificates in the
+      caCertificate attribute.
 
-#### Verification in Case 3
-{: id="sect-4.4.2.3"}
+    1. If a HTTP or FTP server is available, pick the certificates
+      from the "certs-only" CMS message.
+
+    1. If a CMP server is available, request the certificates using
+      the root CA update general message, see {{sect-5.3.19.15}}.
 
-In case 3, the verifier must get access to the new public key of the
-CA.  In case a repository is used, the verifier does the following:
+    1. Otherwise, the get the certificates "out-of-band" using any
```

Comment:
redundant "the", change: "the get the" -> "get the"

Reply:
Thank you, fixed